packages/twenty-front/src/modules/domain-manager/hooks/useDefaultDomain.ts

```diff
@@ -0,0 +1,21 @@
+import { domainConfigurationState } from '@/domain-manager/states/domainConfigurationState';
+import { isMultiWorkspaceEnabledState } from '@/client-config/states/isMultiWorkspaceEnabledState';
+import { useRecoilValue } from 'recoil';
+
+export const useDefaultDomain = () => {
+  const domainConfiguration = useRecoilValue(domainConfigurationState);
+  const isMultiWorkspaceEnabled = useRecoilValue(isMultiWorkspaceEnabledState);
+
+  const defaultDomain = isMultiWorkspaceEnabled
+    ? `${domainConfiguration.defaultSubdomain}.${domainConfiguration.frontDomain}`
+    : domainConfiguration.frontDomain;
+
+  const isDefaultDomain = isMultiWorkspaceEnabled
```

Comment:
use location = useLocation()

Reply:
useLocation doesn't expose the hostname.